### ISSUE TITLE
v2 launch: open the floodgates

### ISSUE THREAD

**Issue by josh-chamberlain** [Police-Data-Accessibility-Project/data-sources-app#544]
### Context

part of #248 
follows #543 

### Requirements 

- #414 
- https://github.com/Police-Data-Accessibility-Project/pdap.io/issues/263
- #299 
- #528 
- #158
- https://github.com/Police-Data-Accessibility-Project/pdap.io/issues/278
- [x] show `make a request` and `follow` buttons to everyone, prompting them to sign up if needed

#### Non-code

- [x] make documentation for the new features: `auth` `follow` `requests`
- [x] prepare a press release + outreach plan

#### Once the above is done
- [x] enable the "sign up" feature flag
- [x] execute the outreach plan

#### Nice-to-have

- https://github.com/Police-Data-Accessibility-Project/pdap.io/issues/253
- https://github.com/Police-Data-Accessibility-Project/pdap.io/issues/203
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/580
  - [ ] enable requests-related feature flags
